frog/imports/client/Wiki/components/TopNavbar/index.js

```diff
@@ -7,8 +7,9 @@ import OverflowPanel from './OverflowPanel';
 
 type TopNavBarPropsT = {
   /** The current Meteor username */
-  user: string,
-
+  username: string,
+  /* Status of the user, username will be displayed in italics if true */
```

Comment:
we prefer // comments unless they are multiline

Reply:
Made the changes